documentation/book/assembly-upgrade-resources.adoc

```diff
@@ -6,7 +6,7 @@
 = {ProductName} resource upgrades
 For this release of {ProductName}, resources that use the API version `{KafkaApiVersionPrev}` must be updated to use `{KafkaApiVersion}`.
 
-The `{KafkaApiVersionPrev}` API version is deprecated in release {ProductVersion}.
+The `{KafkaApiVersionPrev}` API version was deprecated in release 1.2.
```

Comment:
There is not upstream version 1.2. We either need to parametrize this or write it in a way which does not mention the actual release (which might not be so important anyway).

```suggestion
The `{KafkaApiVersionPrev}` API version is deprecated.
```

Reply:
Gargh! I think we can live without a version number. :thumbsup: